<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="HostBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>获取主机信息</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Edit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>271</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="MBtn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>350</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>监听客户端链接</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>端口号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PortEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>360</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="GetText">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>460</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="SendEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>420</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SendBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>420</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发送信息</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
